### PR TITLE
update http_uppercaserer exercise to use const or let instead of var …

### PR DESCRIPTION
**Pull request by iampeterbanjo**
…(#559)

### CODE REVIEW

File: exercises/http_uppercaserer/exercise.js [hunk 1]
```diff
@@ -50,14 +50,14 @@ exercise = comparestdout(exercise)
...
 
   function connect (port, stream) {
-    var input = through2()
-    var count = 0
-    var iv
-    var url = 'http://localhost:' + port
-    var req
+    const input = through2()
+    const count = 0
+    const iv
```
Comment (iampeterbanjo) [inline exercises/http_uppercaserer/exercise.js]: should I add a semi-colons or use comma separated variable declaration?

Comment (AnshulMalik) [reply exercises/http_uppercaserer/exercise.js]: This looks good

File: exercises/http_uppercaserer/exercise.js [hunk 2]
```diff
@@ -50,14 +50,14 @@ exercise = comparestdout(exercise)
...
+    const url = 'http://localhost:' + port
```
Comment (AnshulMalik) [inline exercises/http_uppercaserer/exercise.js]: Let's use template literal here ?

Comment (iampeterbanjo) [reply exercises/http_uppercaserer/exercise.js]: ok

Comment (iampeterbanjo) [reply exercises/http_uppercaserer/exercise.js]: done

File: exercises/http_uppercaserer/exercise.js (hunk 1 above)
Comment (AnshulMalik) [inline exercises/http_uppercaserer/exercise.js]: `iv` should be let

Comment (iampeterbanjo) [reply exercises/http_uppercaserer/exercise.js]: sorry i missed this - I'll fix it

**Comment by iampeterbanjo** [comment]
Travis is failing in Node 4/5 because of

```
/home/travis/build/workshopper/learnyounode/exercises/http_uppercaserer/exercise.js:3
let exercise = require('workshopper-exercise')()
^^^
SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

**Comment by AnshulMalik** [comment]
Hey, @iampeterbanjo  Nice to have you back.

You can just add "use strict" on top of the `exercise.js` file to fix this error :)

**Comment by iampeterbanjo** [comment]
I updated this PR but the build seems to be failing for different reasons now.

**Comment by Nayaco** [comment]
I have received your e-mail.                         Nyanco 33

**Pull request closed without merging**